src/renderer/components/MainPage.tsx

```diff
@@ -483,6 +483,9 @@ class MainPage extends React.PureComponent<Props, State> {
                         developerMode={this.state.developerMode}
                     />
                     {downloadsDropdownButton}
+                    {window.process.platform !== 'darwin' &&
+                        <span style={{width: `${window.innerWidth - (window.navigator.windowControlsOverlay?.getTitlebarAreaRect().width ?? 0)}px`}}/>
+                    }
```

Comment:
Would there be any overlap in full screen due to the condition below?

Reply:
Ugh...why does full screen mode have to exist...
Seems like it's causing some weird behaviour, i'll fix that.

Reply:
Okay I've fixed it for Windows and "fixed" it for Linux.
Unfortunately full screen on Linux really doesn't work very well, so we'll need to address that separately. See: https://github.com/mattermost/desktop/issues/3143